sys/mips/mips/pmap.c

```diff
@@ -2700,7 +2707,12 @@ pmap_copy_page_internal(vm_page_t src, vm_page_t dst, int flags)
 	} else {
 		va_src = pmap_lmem_map2(phys_src, phys_dst);
 		va_dst = va_src + PAGE_SIZE;
-		bcopy((void *)va_src, (void *)va_dst, PAGE_SIZE);
+#if __has_feature(capabilities)
```

Comment:
This looks like an separate fix?

Reply:
Oops, yes, from when I was unsure how tags weren't going missing (turns out the VM means they shouldn't have been going missing, whether or not that's intended). In practice I'm not sure if we hit this case or not (I have not attempted to follow through the logic) but it seems wrong to not handle it properly!

Reply:
Split out into a separate commit (but still part of this PR)